### ISSUE TITLE
Allow decimals in tax calculations

### ISSUE THREAD

**Issue by biilmann** [netlify/gocommerce#153]
We're running into a case where France has a 5.5% tax, and currently all tax settings are read and processed as integers.

We should update the calculator to allow reading taxes with decimals and adjust all calculations accordingly.

**Comment by bcomnes** [comment]
From slack via @mraerino 
> We haven't started on that, since I have been held up by other projects. Though recent refactors of the calculation made this easy to implement

**Comment by mheffner** [comment]
Is this still a high priority issue?

**Comment by mraerino** [comment]
nope. it's still asked for by some people, but we decided to not spend time on it right now. anyone from outside the org is obviously welcome to contribute.

**Comment by Crushla** [comment]
Hello, has this bug been fixed？

**Comment by mraerino** [comment]
this has not been fixed. any contributions are welcome.

**Comment by alecgarza96** [comment]
Is this still an issue?
